packages/playwright/src/reporter.ts

```diff
@@ -242,16 +242,16 @@ class ReplayPlaywrightReporter implements Reporter {
     // skipped tests won't have a reply so nothing to do here
     if (status === "skipped") return;
 
-    const testMetadata = {
+    const testIdData = {
```

Comment:
"metadata" seems more technically correct so I'm curious why this rename?

(No need to change it, just wondering.)

Reply:
we have a lot of different metadata types and I'm getting lost in them ;p this object gets passed to `this.getStepsFromFixture` and `this.getFixtureData. In both of them, the relevant parameter is typed using `TestIdData` type so this rename aligne the name with that.

Reply:
Okay 😅